src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -79,7 +79,6 @@ def create_existing_high_priority_issue_data_condition(
 def create_event_attribute_data_condition(
     data: dict[str, Any], dcg: DataConditionGroup
 ) -> DataCondition:
-    # TODO: Add comparison validation (error if not enough information)
```

Comment:
these are already accounted for via `comparison_json_schema` in `DataConditionHandler`